source/en/components/pageview.md

```diff
@@ -0,0 +1,114 @@
+# PageView component reference
+
+The Page view, Use when you need to switch the entire page.
+
+![pageview-inspector](./pageview/pageview-inspector.png)
+
+Click the 'add component' pageView at the bottom of the **Properties** panel and select 'PageView' from 'add UI component'. You can then add the PageView component to the node.
+
+Please refer to the script interface of the pageView [PageView API](../api/classes/PageView.html)
+
+## PageView attribute
+
+| Attribute                | Function description |
+| --------------           | ----------- |
+| SizeMode                 | Specify the size type of each page in PageView |
+| Content                  | It is a node reference that is used to create the contents of the PageView |
+| Direction                | The page view direction |
+| ScrollThreshold          | The scroll threshold value, when drag exceeds this value, release the next page will automatically scroll, less than the restore |
```

Comment:
This value will be multiplied with the distance between two pages, to get the threshold distance. If user scroll distance is larger than this threshold distance, the page will turn immediately.